graylog2-web-interface/src/components/indices/IndexSetFieldTypes/IndexSetFieldTypesList.test.tsx

```diff
@@ -273,7 +273,7 @@ describe('IndexSetFieldTypesList', () => {
       await screen.findByRole('heading', { name: /Remove field type overrides/i });
       await screen.findByText('Rotate affected indices after change');
 
-      const modal = await screen.findByRole('dialog');
+      const modal = await screen.findByRole('dialog', { name: /Remove field type overrides/i });
```

Comment:
Do you think it is overkill to implement a shared function for the modal query which can be reused in all related tests?